includes/functions.php

```diff
@@ -1471,11 +1518,14 @@ function tsml_import_buffer_set($meetings, $data_source_url = null, $data_source
     $user_id = get_current_user_id();
 
     //convert the array to UTF-8
-    array_walk_recursive($meetings, function (&$item) {
-        if (!mb_detect_encoding($item, 'utf-8', true)) {
-            $item = utf8_encode($item);
-        }
-    });
+    if (function_exists('mb_detect_encoding')) {
```

Comment:
I think I didn't have this PHP lib installed an so, just wrapping with a check. Or maybe Wayne had this in there. I don't know anymore, Git blame should know.

Reply:
we did recently encounter a user who didn't have the `mb_` functions enabled and started checking

Reply:
Hmmm. I don't remember adding it. Think it came in on a merge after a patch release, which aligns with what Josh said.